### ISSUE TITLE
[deeplab]NotFoundError (see above for traceback): Key aspp1_depthwise/BatchNorm/beta not found in checkpoint

### ISSUE THREAD

**Issue by jiyongma** [tensorflow/models#3983]
when I run 
python deeplab/eval.py \
    --logtostderr \
    --eval_split="val" \
    --model_variant="xception_65" \
    --atrous_rates=6 \
    --atrous_rates=12 \
    --atrous_rates=18 \
    --output_stride=16 \
    --decoder_output_stride=4 \
    --eval_crop_size=513\
    --eval_crop_size=513 \
    --dataset="ade20k" \
    --checkpoint_dir="./deeplab/datasets/ADE20K/exp/train_on_train_set/train" \
    --eval_logdir="./deeplab/datasets/ADE20K/exp/train_on_train_set/eval" \
    --dataset_dir="./deeplab/datasets/ADE20K/tfrecord"

NotFoundError (see above for traceback): Key aspp1_depthwise/BatchNorm/beta not found in checkpoint
	 [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]
	 [[Node: save/RestoreV2/_299 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_306_save/RestoreV2", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]

**Comment by karmel** [comment]
I apologize, but I am having a hard time understanding what the problem is, where the problem is, and what version it affects. Please update and pay attention to the issue template (https://github.com/tensorflow/tensorflow/issues/new). Please provide all the information it asks. Thank you.

**Comment by cfosco** [comment]
Hi @jiyongma and @karmel , 
I am having the same issue. To answer @karmel 's questions,  this problem occurs when trying to fine-tune deeplabv3+, with tensorflow 1.6 or 1.7, and cudnn 7.0.4. The issue is triggered when running the eval.py script. I gave a thorough description of the issue and the code modifications that I did to try to train on a custom dataset on issue #3992 . Maybe @aquariusjay, @YknZhu or @walkerlala have an answer to this?

**Comment by cuicuizhang1989** [comment]
may be your log dir has different kind of checkpoint such as xception_65 and mobilenetv2 both in the save dir.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
